src/main/java/luke/Luke.java

```diff
@@ -0,0 +1,301 @@
+package luke;
+
+import luke.exceptions.IncorrectInput;
+import luke.exceptions.InsufficientArguments;
+import luke.exceptions.InvalidCommand;
+import luke.exceptions.LukeException;
+import luke.tasks.Deadline;
+import luke.tasks.Event;
+import luke.tasks.Task;
+import luke.tasks.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Luke {
+
+    private static final String LOGO =
```

Comment:
Very good work in using constants to prevent the use of magic strings and numbers!